About.html

```diff
@@ -0,0 +1,141 @@
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li><button class="nav-btn"><a href="#">CC campaign</button></a></li>
+           </ul>
+           </div>
+         </nav>
+        </header>
+ <section>
+  <div>
+    <p class="about-slogan">"Hello! Sharing world"</p>
+  </div>
+  <div>
+  <h1 class="about-name">CREATIVE COMMONS <br>
+      GLOBAL SUMMITY 2015
+  </div>
+  <hr class="line">
+  <p class="about-para">
+    The Globle Summit brings together the community og experts academics, and activists who comprise 
+    the Creative Commons affilate network in a different country every two year.
+    This year, we hope to expand our invitation list including organizations and individuals 
+  </p>
+
+  <p class="about2">
+    The Globle Summit brings together the community og experts academics, and activists who comprise 
+    the Creative Commons affilate network in a different country every two year.
+    This year, we hope to expand our invitation list including organizations and individuals who wants to work with us on share projects that advance thhe cause of the Commons, free culture and open knoledge.
+    so if you're active and engaged in the worlds of open contet and knowledge- free software advocates, wikipedians, open knoledge galleries, libraries, museums, archieves, governments and foundations, lawyers, and activists, we hope you'll consider joining us this year to build strong, more vibrant commons together
+  </p>
+      <p class="desk-about">
+        please contact us per Email for any further question <br>
+          about CC Global Summit 20215 <br>
+          <span class="cc-Email">ccsummit@2015@cckorea.org</span>
+      </p>
+      </section>
+    </div>
+
+      <div>
+            <p class=" creative">
+            Creative Commons Global Summit 2015 Logo<br>
+              The Logo of CC Global summit 2015 was Decided 
+              through the logo competition from 8.June to July.
+            </p>
+            <p class="creative-desk">
+              <span class="heading">Creative Commons Global Summit 2015 Logo</span>  <br>
+              <span class="last-head">The Logo of CC Global summit 2015 was Decided 
+              through the logo competition from 8.June to July. <br>
+              After noline public voting and selection committee judging process, the logo from Neseh Agrawal won the prize. <br>
+              after consultation with Mr. agrawal we optionalized the prize winning logo for this year summit as following </span>
+            </p>
+          <div>
+            <img  class="about-logo" src="logos/logo.png" alt="logo-cc">
+          </div>
+          
+<p class="about-past">
+  See the past CC Global Summits
+  </p>
+  <hr class="line">
+  <p class="last-para">
+    Take a look at the two CC Global Summit s which took place 
+  in Beunos Aires and in Warsaw.
+  </p>
+
+</p>
```

Comment:
- [ ] Kindly add images for this section like in the original design

![image](https://github.com/mumbereh/Capston-project/assets/98453979/e5a687dd-1b2a-440a-8e30-15c1bfa780f8)

### Checkout the original design

![image](https://github.com/mumbereh/Capston-project/assets/98453979/271ca2f6-fa3c-4e02-acb1-190739339153)